aci/resource_aci_snmpcommunityp.go

```diff
@@ -103,7 +116,8 @@ func resourceAciSNMPCommunityCreate(ctx context.Context, d *schema.ResourceData,
 	if Name, ok := d.GetOk("name"); ok {
 		snmpCommunityPAttr.Name = Name.(string)
 	}
-	snmpCommunityP := models.NewSNMPCommunity(fmt.Sprintf("community-%s", name), VRFSNMPCtxDn, desc, nameAlias, snmpCommunityPAttr)
+
+	snmpCommunityP := models.NewSNMPCommunity(fmt.Sprintf(models.RnsnmpCommunityP, name), SNMPPolicyDn, desc, nameAlias, snmpCommunityPAttr)
```

Comment:
what's this fmt.Sprintf(models.RnsnmpCommunityP, name)?

Reply:
models.RnsnmpCommunityP is the same as "community-%s" but it is defined in the aci-go-client so we avoid repeating it and only have to update it once if we need to.